adversarial-thinking.adoc

```diff
@@ -83,19 +83,19 @@ conservative
 ____
 
 He refers to these simplistic narratives as monomania, meaning that by
-focusing only on a single thing, e.g., "just HODL", you stop thinking
+focusing only on a single thing, e.g. "just HODL", you stop thinking
 about important stuff like keeping your Bitcoin secure or doing your
-best to use Bitcoin in a trustless manner.
+best to use Bitcoin in a trustless manner.[comm: this section is a bit obscure to me. I undestand everything that is said but I don't quite get the logic behind it: what does the title refer to? And if we open by saying that adversarial thinking is relevant not only to bitcoiners, why do we only talk about Bitcoin then? And what is the logical connection between the over-simplistic narratives and thinking adversarially? I imagine the answers to all these questions but I feel like they should be more explicitly outlined in the text]
 
 === Threats
 
 There are a lot of known weaknesses in Bitcoin, and many of them are
-actively being exploited. To get a glimpse of that, have a look at
-https://en.bitcoin.it/wiki/Weaknesses[the Weaknesses page] on the
-Bitcoin wiki. They mention a wide variety of problems, for example,
+actively being exploited. To get a glimpse of that, have a look at the
+https://en.bitcoin.it/wiki/Weaknesses[Weaknesses page] on
+Bitcoin wiki. There are mentioned a wide variety of problems, such as
 wallet theft and denial-of-service attacks.
 
-[quote, Various authors, The Bitcoin wiki]
+[quote, Various authors, Bitcoin wiki]
```

Comment:
This article has evolved over a long time, so I guess your're right to not set a year on this one. https://en.bitcoin.it/w/index.php?title=Weaknesses&offset=&limit=500&action=history

Reply:
Yeah, these ones are hard. You can put a date on a speech or an article but not on an encyclopedic definition. I think it's the standard to leave it undated when quoting this kind of sources